app/views/step_by_step_pages/index.html.erb

```diff
@@ -1,2 +1,59 @@
-<h1>StepByStepPages#index</h1>
-<p>Find me in app/views/step_by_step_pages/index.html.erb</p>
+<%
+  links = [
+    {
+      text: 'Step by step publisher',
+    }
+  ]
+%>
+
+<%= render 'partials/step_breadcrumb', links: links %>
+
+<h1>Step by step navigation publisher</h1>
+<p class="text-right">
+  <%= link_to 'Create new', new_step_by_step_page_path, class: "btn btn-primary" %>
+</p>
+
+<table class="table table-striped">
+  <thead>
+    <tr>
+      <th>Title</th>
+      <th colspan="6"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @step_by_step_pages.each do |step_by_step_page| %>
+      <tr>
+        <td>
+          <%= step_by_step_page.title %>
+        </td>
+        <td>
+          <%= link_to 'Edit content', step_by_step_page %>
+        </td>
+        <td>
+          Edit rules
+        </td>
+        <td>
+          <%= link_to("Preview", step_by_step_page_preview_path(step_by_step_page)) %>
+        </td>
+        <td>
+          Download JSON
+        </td>
+        <td>
+          <%= link_to 'Delete', step_by_step_page, method: :delete, data: { confirm: 'Are you sure?' } %>
+        </td>
+        <td>
+          <span class="label label-success">Status</span>
+        </td>
+      <% if false %>
```

Comment:
I guess this is the equivalent of a comment, which I would probably remove until needed.